### DISCUSSION TITLE
v0.9.0

### DISCUSSION THREAD

**Discussion by lxndrblz** [lxndrblz/anatole#179] [Releases]
# New Features
* Added Korean and Arabic Translation
* Added RTL Support
* User defined Post section names
* Allow multiline slogans in the sidebar

# Fixes
* Improved the overall appearance to closer resemble the original theme by CaiCai
* Resolved an issue where log links would overflow the page width
* Darkened the dark mode
* Changed URLs to relative URLs
* Allow "." and "#" signs in tags and categories 

<hr /><em>This discussion was created from the release <a href='https://github.com/lxndrblz/anatole/releases/tag/v0.9.0'>v0.9.0</a>.</em>

**Comment by nicholascarey** [comment]
How do you check which version of the theme you are currently on, and how do I apply the update? I ran the command in the README but nothing seemed to happen, and there was nothing to commit. 

I'm working in RStudio. 

Thanks.

**Reply by lxndrblz** [reply]
@nicholascarey If you added the theme as a submodule, you should be able to run a git `submodule update --remote --merge` in the directory where you had initialized to update your theme. If you had added the theme as a submodule you can run a `git submodule status` to get the current version tag e.g., 0.8.0.

If you cloned/downloaded the theme you will have to clone it once more.

I am not too familiar with using blogdown/R. Based on [this](https://masalmon.eu/2020/02/29/hugo-maintenance/) blog you should be able to force an update by running blogdown::install_theme() with force = TRUE or by replacing anatole in the theme folder as explained above.

**Comment by nicholascarey** [comment]
Ok, thanks. Yes, `blogdown::install_theme(force = TRUE)` worked. The submodule commands did not, even in the correct directory, so i guess i haven't installed it as a submodule.... 

If you are interested in feedback - i prefer the previous version of the theme, so actually I won't be updating my site to use this particular version. I prefer the previous font, and I don't believe dark modes need to be completely black. I think it works against readability. Maybe have it as an option in the yaml? light/dark/black?

Cheers

**Reply by lxndrblz** [reply]
@nicholascarey  Thanks for your feedback and sharing your concerns. I highly appreciate it and will consider it for future changes and revisions of this theme. 

The reason I had updated the font back to a sans-serif font is because I wanted the theme look closer to the original theme and also look more minimalistic. The same goes for the background colour. Personally, I found the greyish to appear blurred out. I'd also looked at various other dark modes (like here on GitHub) and noticed that most sites choose a darker colour than what was configured previously. However, I have to agree that it takes some time to adjust and readability might be impared.

If you want to stay up to date, but use a custom dark theme and font, then I can encourage you to use a custom CSS file, which overrides the variables for the colors. Please let me know if you need support setting one up.
